lib/turn.rb

```diff
@@ -61,10 +61,19 @@ def award_spoils(winner)
     end
   end
 
+  #Helper Methods
+  def zero_index_rank(player)
```

Comment:
Made helper methods to clean up the code a bit. Before the helper methods, lines 22 and 23 were way over the 80 character limit.